portfolio/src/main/webapp/scripts/views/comments.js

```diff
@@ -67,6 +67,7 @@ const createCommentContainer = (comment) => {
 
 /** @param {Array.object} comments responded objects from API */
 const renderCommentList = (comments) => {
+  comments.sort((x, y) => y.timestamp - x.timestamp) // reverse order
```

Comment:
it's not intuitively clear what "reverse" means in the context of timestamps.  Latest to earliest?  Earliest to latest?  Just put one of those to make it clearer.